src/App.tsx

```diff
@@ -25,39 +26,50 @@ const Logs = React.lazy(() => import('./pages/Logs'));
 const App: React.FC = () => {
     return (
         <Suspense fallback={<Skeleton animation="wave" />}>
-            <Routes>
-                {
-                    // TODO - do we need the default path to use a "redirector"/"landing" page kind of thing?
-                }
-                <Route path="" element={<Login />} />
-                <Route path="login/*">
+            <CheckAuth>
```

Comment:
This is the  real change here. We want to check if the user is already logged in way up in the stack so we can handle stuff early. This is basically just a "is there a user in local storage? Then make sure they're logged in"